Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -118,7 +118,9 @@ - (void)viewDidLoad
     } else {
         [self.view addGestureRecognizer:self.longPressGestureRecognizer];
     }
-
+    if (@available(iOS 11.0, *)) {
```

Comment:
Nitpick, but could you add a blank line above and below this `if`, just to help readability?